### PR TITLE
Disable Bitcode for Xcode 8

### PR DESCRIPTION
**Pull request by colemancda**

### CODE REVIEW

File: CryptoSwift.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1281,11 +1283,11 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				APPLICATION_EXTENSION_API_ONLY = YES;
-				BITCODE_GENERATION_MODE = marker;
 				CLANG_ENABLE_MODULES = YES;
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				ENABLE_BITCODE = NO;
```
Comment (krzyzanowskim) [inline CryptoSwift.xcodeproj/project.pbxproj]: hm... why?

Comment (colemancda) [inline CryptoSwift.xcodeproj/project.pbxproj]: https://github.com/krzyzanowskim/CryptoSwift/issues/240#issuecomment-233115303

Comment (krzyzanowskim) [inline CryptoSwift.xcodeproj/project.pbxproj]: this shouldn't be the case here. OpenSSL thing is different.

**Comment by krzyzanowskim** [comment]
- ErrorProtocol -> Swift.Error change is post Xcode 8 beta 3 (and is available on swift3-snapshots) branch.

- When it comes to disabling bitcode... I don't want to merge that change. I believe this is temporary problem that will be fixed and I don't want to forget to re-enable it.

**Pull request closed without merging**